File: app/src/main/java/com/example/androidscript/Menu/Basic/BasicScriptCompiler.java
package com.example.androidscript.Menu.Basic;

import com.example.androidscript.util.ScriptCompiler;
import com.example.androidscript.util.DebugMessage;
import com.example.androidscript.util.FileOperation;

import java.util.Vector;

public class BasicScriptCompiler extends ScriptCompiler {
    @Override
    public void compile(Vector<Vector<String>> Data) {
        FileOperation.writeWords(BasicEditor.FolderName + "Run.txt", Data);
    }
}

File: app/src/main/java/com/example/androidscript/activities/basic/BasicScriptCompiler.kt
package com.example.androidscript.activities.basic

import com.example.androidscript.util.FileOperation
import com.example.androidscript.util.ScriptCompiler
import java.util.*

class BasicScriptCompiler : ScriptCompiler() {
    override fun compile(data: Vector<Vector<String>>) {
        FileOperation.writeWords(BasicEditor.folderName + "Run.txt", data)
    }
}